packages/strapi-parts/src/Accordion/Accordion.stories.mdx

```diff
@@ -19,7 +20,17 @@ This is the doc of the `Accordion` component
 
 ## Accordion
 
-Description...
+Accordions are collapsable blocks that vertically hide a stack of content. Clicking on the block reveals the associated content. 
+
+**Best practices**
+
+- Use accordion when you need big and/or related chunks of content to be organized in a same place.
+- Use accordion for lower priority information.
+- Use to shorten pages and increase readability.
+- The chevron icon pointing down indicates a collapsed state and pointing up indicates an expanded state.
+- Hover and active states should be used anytime it's possible.
```

Comment:
this is handled under the hood, people won't have to manage this

Reply:
Don't you think it's preferable to add this anyway so users can understand the good practices?

Reply:
We can leave it, but the user won't have any freedom in dealing with it, since it's part of the component 😊